queries/meson/highlights.scm

```diff
@@ -36,6 +36,7 @@
   "-="
   ">"
   ">="
+	"?"
 ] @operator
```

Comment:
There was a discussion in the past on whether ternary should be `@operator` or `@conditional`. The `@conditional` side won (see C/C#/Java). I also preferred `@operator` back then. But we should keep consistency.

 We now query fallbacks. So now a `@conditional.ternary` would be possible if applied to consistently across languages

Reply:
So when will this keyword be available?

Reply:
Strange, the check was intended to allow everything that starts with `@conditional.Xxxx` let me check what's wrong